macosx/VDKQueue/VDKQueue.mm

```diff
@@ -171,22 +169,23 @@ - (VDKQueuePathEntry*)addPathToQueue:(NSString*)path notifyingAbout:(u_int)flags
 
 - (void)watcherThread:(id)sender
 {
-    int n;
-    struct kevent ev;
+#if DEBUG_LOG_THREAD_LIFETIME
+    NSLog(@"watcherThread started.");
```

Comment:
nit: One thing I would suggest is to append some number/id/etc. to watcherThread to differentiate between many instances of `watcherThread` or `VDKQueue`.

Reply:
Thread number is already always displayed:
<img width="338" alt="Screenshot 2023-11-28 at 09 42 35" src="https://github.com/transmission/transmission/assets/839992/576564cb-12c2-4264-923d-562fee9434c0">

Reply:
Yeah, I know.
But you are showing debugger UI, but this message is logged.
I think it's no longer an issue with modernized logs in Xcode 15.
So it's a nit :)

Reply:
I see, but that log message was already like that (this is not something that I changed in this PR). And I don't think anyone would need multiple watcher threads since that would multiply the activity without offering benefits.